glyphs-reader/src/font.rs

```diff
@@ -67,15 +67,33 @@ pub struct Glyph {
     pub other_stuff: BTreeMap<String, Plist>,
 }
 
+/// Represents a path OR a component
+///
+/// <https://github.com/schriftgestalt/GlyphsSDK/blob/Glyphs3/GlyphsFileFormat/GlyphsFileFormatv3.md#differences-between-version-2>
+#[derive(Clone, Debug, FromPlist, PartialEq, Eq, Hash)]
+pub struct Shape {
+    // When I'm a path
+    pub closed: Option<bool>,
+    pub nodes: Option<Vec<Node>>,
+
+    // When I'm a component I have a ref
+    // but it seems to end up other_stuff because r#ref doesn't work
```

Comment:
FWIW, in Glyphs.app's python scripting API (as well as glyphsLib.classes), a GSComponent has a `name` attribute that contains this `ref`

https://docu.glyphsapp.com/#GSComponent.name

Reply:
oh they renamed that component attribute in the plist from `name` (v2) to `ref` (v3), I wonder why..

Reply:
Now that it ends up in Component we have a glyph_name.